client/style.css

```diff
@@ -264,27 +265,38 @@ h1{
 .ig-thumbnail-inner-container {
   position: relative;
   overflow: hidden;
-  height: 100%;
+  height: 96%;
 }
 
 .igtc-sliding-inner-div {
   position: relative;
   height: 100%;
   transition: all 0.5s;
+  cursor: default;
+}
+
+.one-percent-high-shim {
```

Comment:
Another way you could do it is to make this a margin-top or margin-bottom instead of height, which would let you just add the calss as a token to anything where you want spacing above/below it.